### DISCUSSION TITLE
Storage Error: Tag prop not found

### DISCUSSION THREAD

**Discussion by KieuVanHanh** [vesoft-inc/nebula#5986] [Q&A]
after I run the command
ALTER TAG Ahihi CHANGE (officialGroup BOOL DEFAULT FALSE NOT NULL);

I get an error when Storage Error: Tag prop not found when querying
Match(u:Ahihi ) return u.Ahihi.officialGroup

**Comment by xiajingchun** [comment]
1. what's the nebula version did you use?
2. what's original data type of `officialGroup`?
thx.
